docs/src/containers/home/info-tiles.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+  import * as attractions from 'attractions';
+  import { s } from 'attractions/utils';
+  import { GridIcon, Edit2Icon, FeatherIcon } from 'svelte-feather-icons';
+  import InfoTile from 'src/components/home/info-tile.svelte';
+
+  const totalComponents = [...Object.keys(attractions)].length;
```

Comment:
Why the spread and array reconstruction?
```suggestion
  const totalComponents = Object.keys(attractions).length;
```
should suffice